1.9/administration/securing-your-cluster.md

```diff
@@ -35,37 +35,11 @@ By default, Admin Router will permit unencrypted HTTP traffic. This is not
 considered secure, and you must provide a valid TLS certificate and redirect
 all HTTP traffic to HTTPS in order to properly secure access to your cluster.
 
-Once you have a valid TLS certificate, install the certificate on each master.
+After you have a valid TLS certificate, install the certificate on each master.
 Copy the certificate and private key to a well known location, such as under
-`/etc/ssl/certs`. It's suggested that you change the permissions of the
-private key to not be world-readable. Edit the Admin Router `nginx.conf`, by
-opening the file located at
-`/opt/mesosphere/active/adminrouter/nginx/conf`. Modify the nginx
-configuration `server` section, to look like this:
-
-```
-  server {
-    listen 80 default_server;
-    server_name _;
-    return 301 https://$host$request_uri;
-  }
-
-  server {
-      listen 443 ssl spdy default_server;
-      ssl_certificate /etc/ssl/certs/your-cert.crt;
-      ssl_certificate_key /etc/ssl/certs/your-key.key;
-
-      # rest of config intentionally omitted
-```
-
-In the block above, we're adding a new `server` section above the existing one
-which will redirect all HTTP traffic to HTTPS. We've also removed the
-`listen 80 ...` directive from the second server section. After making the
-changes, restart Admin Router:
-
-```console
-# systemctl restart dcos-adminrouter
-```
+`/etc/ssl/certs`. 
+
+It's recommended that you run HAProxy in front of Admin Router. For more information, see the [documentation](/docs/1.9/administration/haproxy-adminrouter/).
```

Comment:
`If you run HAProxy in front of Admin Router, you should secure the communication between them. For information about securing your communication, see the [documentation](/docs/1.8/administration/haproxy-adminrouter/).`